database/database.json

```diff
@@ -1,10 +1,10 @@
 {
 	"dev": {
 		"driver": "pg",
-		"user": "postgres",
-		"password": "opensesame",
-		"host": "localhost",
-		"database": "cyf_tms",
-		"port": "5432"
+		"user": "${DB_USER}",
+		"password": "${DB_PASS}",
+		"host": "${DB_HOST}",
+		"database": "${DB_NAME}",
+		"port": "${DB_PORT}"
```

Comment:
We need to do this for production. For the dev environment, we can leave it the way it was before